test/helpers/envtest.go

```diff
@@ -91,21 +119,78 @@ type TestEnvironment struct {
 
 	doneMgr chan struct{}
 }
+type WebhookSupportingType interface {
```

Comment:
If I'm not wrong, this type is not used anywhere. Remove?

Reply:
Yes, you are right. Will be removed